test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/XmlDataContractSerializerInputFormatterTest.cs

```diff
@@ -293,17 +292,14 @@ public void SetMaxDepth_ThrowsWhenMaxDepthIsBelowOne()
             Assert.True(context.HttpContext.Request.Body.CanRead);
         }
 
+        [ReplaceCulture]
         [ConditionalFact]
         // Mono issue - https://github.com/aspnet/External/issues/18
         [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
```

Comment:
Odd this test was double-protected. This attribute and the comment above can also be removed. Same just a bit later in this file.